### ISSUE TITLE
Issue when updating Login email

### ISSUE THREAD

**Issue by Davinkjohnson** [episphere/connect#660]
This was in STAGE. Possible issue, check logs and see if we can re-create. 

Scenario: Pt has email address as current sign-in method, update to new email address, and then add account phone #.
When one of our testers tried to log into their MyConnect account with their new account email after adding an account phone, they received an “action code invalid error” (screenshot “2” attached). She selected “dismiss” and tried signing in with the email again and did not encounter the same error message.

**Comment by Davinkjohnson** [comment]
![2_Tina_action code invalid with email after adding phone number](https://github.com/episphere/connect/assets/87718852/287fc548-aac0-4817-8585-887044de0c06)

**Comment by JoeArmani** [comment]
I attempted to reproduce this process 3x as described with a new participant at https://myconnect-stage.cancer.gov/. I was unable to reproduce the issue.

That doesn't mean it isn't occurring, but any time I've come across this issue in development, it has been due to one of the following:
(1) using an old action link
(2) accidentally using an action link from the previous account in testing
(3) accidentally double-clicking the action link. This link automatically invalidates after the first click, so even a nearly-immediate 2nd try will produce the 'action code invalid' error, which is a security mechanism that firebase auth has in place.

If we can find a reproducible version of this issue that isn't the intended purpose, we should debug it. As it stands, I don't see an action to be taken since it isn't reproducible thus far.
